saike_midi_arp/saike_midi_arp.jsfx

```diff
@@ -1,8 +1,8 @@
 desc:Saike MIDI ARP (beta)
 tags: midi arpeggiator
-version: 0.34
+version: 0.35
 author: Joep Vanlier
-changelog: Bump version number.
+changelog: When set to MIDI mode, reset the pattern only after all previous notes are relased.
```

Comment:
Minor nitpick, but there's a typo. Should be `released`.

Reply:
My apologies😅, I see the PR is already merged/closed but let me know if you want me to fix it.
Thanks for merging these changes!